src/style/_loginform.scss

```diff
@@ -0,0 +1,55 @@
+@import "./variables.scss";
+.login-part{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-family:$comfortaa;
+}
+h1 {
```

Comment:
this will effect every h1 in the app it needs to be nested or more specific.

Reply:
got it ill change it